Source/Widgets/Viewer/Viewer.js

```diff
@@ -524,14 +580,46 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
                     timeline.resize();
                 }
             });
-        } else if (defined(timeline)) {
-            timeline.container.style.right = 0;
+        }
+
+        // VR
+        var vrButton;
+        var vrSubscription;
+        var vrModeSubscription;
+        var vrCallback;
+        if (options.vrButton === true) {
```

Comment:
Why is every other boolean checked with `!== false`? Which is what I meant to put to be consistent with the other checks.